src/config/REVO_AT/config.h

```diff
@@ -19,7 +19,7 @@
  * If not, see <http://www.gnu.org/licenses/>.
  */
 
-#define FC_TARGET_MCU     AT32F435
+#define FC_TARGET_MCU     AT32F435_4M
```

Comment:
This needs to be an actual target location.

Target/AT32F435_4M/x will need to be created.

Reply:
Yes, I understand that. I’m the only person with a REVO_AT whereas a number of people have the NEUTRONRCF435, so this is just a placeholder.

Reply:
Now fixed. `config.h` can specify the following for 4M devices

```
#define FC_TARGET_MCU     AT32F435M
```

or for 1M devices

```
#define FC_TARGET_MCU     AT32F435G
```